README.md

```diff
@@ -90,6 +90,62 @@ Looking for inspiration? You can view the following profiles for an example:
 - [Krupali Trivedi || Chai](https://github.com/EddieHubCommunity/LinkFree/blob/main/data/krupalitrivedi.json)
 - [Pradumna Saraf](https://github.com/EddieHubCommunity/LinkFree/blob/main/data/Pradumnasaraf.json)
 
+
+<!-- Testimonials STARTs Here -->
+
+
+## ⛩️ Testimonials
+
+
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Here are some testimonials from individuals who have used LinkFree:<br><br>
```

Comment:
why `<br>`?

Reply:
why `&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;`?

Reply:
please just use standard markdown

Reply:
I used `<br>`  for inserts a single line break gives spaces for section and  `&nbsp;`  for  creating spaces in-front of line for alignment ...

Reply:
For single-line breaks, you can use `\` at the end of a sentence.